index.js

```diff
@@ -104,6 +127,17 @@
      */
     disableVisbilityHandling: function() {
       if (this._rcv_fn) {
+        if (!this._dom_node) {
```

Comment:
disable only runs its code if `this._rcv_fn` exists, and if it does, `this._dom_node` also exists, so we can do without L130-133.